uelc/main/tests/test_models.py

```diff
@@ -1,6 +1,63 @@
 from django.test import TestCase
+#from django.core.exceptions import ValidationError
```

Comment:
In general, commented out code is considered bad practice. Here's the whys/wherefores: http://www.markhneedham.com/blog/2009/01/17/the-danger-of-commenting-out-code/.